cranelift/filetests/filetests/stackmaps/incoming_args.clif

```diff
@@ -0,0 +1,81 @@
+test stackmaps
+set enable_safepoints=true
+target x86_64
+
+;; Incoming args get included in stack maps.
+
+function %incoming_args(r64, r64, r64, r64, r64) -> r64 windows_fastcall {
+; check:  r64 [32]
+; nextln: ss0 = incoming_arg 8, offset 32
+; nextln: ss1 = incoming_arg 24, offset -24
+; nextln: ss2 = explicit_slot 32, offset -64
+
+    fn0 = %none()
+; nextln: sig0 = () fast
+; nextln: fn0 = %none sig0
+
+block0(v0: r64, v1: r64, v2: r64, v3: r64, v4: r64):
+; check: v4: r64 [ss0]
+
+    call fn0()
+; check:  safepoint v4
+; nextln: call_indirect
+    return v4
+}
+
+; check:  Stack maps:
+; nextln: 
+; nextln: safepoint v4
+; nextln:   - mapped words: 13
+; nextln:   - live: [12]
+
+;; Outgoing args are not included in stack maps.
+
+function %outgoing_args(r64, r64, r64, r64, r64) -> r64 {
+; check:  function %outgoing_args
+; nextln:     ss0 = outgoing_arg 8, offset 32
+; nextln:     ss1 = spill_slot 8, offset -32
+; nextln:     ss2 = incoming_arg 24, offset -24
+
+    fn0 = colocated %f(r64, r64, r64, r64, r64) windows_fastcall
+; nextln:     sig0 = (r64 [%rcx], r64 [%rdx], r64 [%r8], r64 [%r9], r64 [32]) windows_fastcall
+; nextln:     fn0 = colocated %f sig0
+
+block0(v0: r64, v1: r64, v2: r64, v3: r64, v4: r64):
+; check: block0(v6: r64
+; check: ss1]         v0 = spill v6
+
+    call fn0(v0, v1, v2, v3, v4)
+; check:  safepoint v0, v1, v2, v3, v5, v7
+; nextln: call fn0
+
+    return v0
+; check: v8 = fill v0
+; check: return v8
+}
+; check:  Stack maps:
+; nextln: 
+; nextln: safepoint v0, v1, v2, v3, v5
+; nextln:   - mapped words: 10
+; nextln:   - live: [6]
+
+; check: TODO FITZGEN: this stack map and stack frame don't make sense:
+;
+; 1. `mapped words` seems like it should be 9, not 10?
+;
+; 2. `live: [6]` seems like it should be `live: [0]`, since `v0` is stored in
+;    `ss1`?
+;
+; Here is a table of the stack frame and its slots:
+;
+; Frame Base Offset | Stack Slot | N^th Word in Stack Map
+; ------------------+------------+-----------------------
+;                32 | ss0        | 8
+;                24 | -          | 7
+;                16 | -          | 6
+;                 8 | -          | 5
+;                 0 | -          | 4
+;                -8 | ss2        | 3
+;               -16 | ss2        | 2
+;               -24 | ss2        | 1
+;               -32 | ss1        | 0
```

Comment:
This is the function that looks like it has an incorrect stack frame and stack map. See the comment and table for explanation.